app/src/main/java/com/thenewboston/kotlinsdk/home/viewmodels/BankViewModel.kt

```diff
@@ -1,13 +1,45 @@
 package com.thenewboston.kotlinsdk.home.viewmodels
 
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
 import com.thenewboston.kotlinsdk.LIST_DATA_LIMIT_DEFAULT
 import com.thenewboston.kotlinsdk.home.repository.bank.BankRepo
+import com.thenewboston.kotlinsdk.network.models.BankConfigModel
+import com.thenewboston.kotlinsdk.network.models.GenericListDataModel
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers.IO
+import kotlinx.coroutines.launch
 
 class BankViewModel(
     private val repo: BankRepo
 ) : ViewModel() {
-    suspend fun getBankConfig() = repo.getBankConfig()
-    suspend fun getNumOfConfServices() = repo.getNumOfConfServices()
-    suspend fun getBankAccounts(limit: Int = LIST_DATA_LIMIT_DEFAULT, offset: Int) = repo.getAccountsForBanks(limit, offset)
+    /* Overview */
+    private val mLiveDataBankOverview = MutableLiveData<Pair<String?, BankConfigModel?>>()
+    val liveDataBankOverview : LiveData<Pair<String?, BankConfigModel?>> = mLiveDataBankOverview
+
+    fun getBankConfig() {
+        CoroutineScope(IO).launch {
```

Comment:
Pro tip: Always use the view model's scope as it will handle the cancellation for you. You can use the view model kts extensions for this.

`viewModelScope.launch {
 //do your request here
}`